Cabal/Distribution/Simple/Build.hs

```diff
@@ -19,13 +19,13 @@
 --
 
 module Distribution.Simple.Build (
-    build, repl,
+    build, showBuildInfo, repl,
     startInterpreter,
 
     initialBuildSteps,
     createInternalPackageDB,
     componentInitialBuildSteps,
-    writeAutogenFiles,
+    writeAutogenFiles
```

Comment:
What's up whit these changes? Isn't writeAutogenFiles supposed to go into a different PR?

Reply:
This isnt actually a change that has been introduced by this PR. I thought this is related to write-autogen-files, but as far as I can tell, this is not related to this pr.